test/helpers.test.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Normalize various items in the unit tests, such as:
+ *  - line endings
+ *  - path separators
+ *  - ascii symbols (checkmark, x, etc...).
+ */
+export function normalize(text: string) {
```

Comment:
This is where most of this PR magic happens. It normalizes newlines, path separators, various icons, etc.